packages/docs/src/components/docsearch/doc-search.tsx

```diff
@@ -34,10 +36,10 @@ export type DocSearchState = {
   status: 'idle' | 'loading' | 'stalled' | 'error';
   initialQuery?: string;
 };
-
 export interface DocSearchProps {
   appId: string;
   apiKey: string;
+  isOpen: Signal<boolean>;
```

Comment:
small nitpick: can you rename this to `bind:open`? That's consistent with Qwik UI and indicates it's a Signal.